README.md

```diff
@@ -63,6 +63,15 @@ and the following CPU types:
 * Storage Admin
 * Vertex AI Administrator
 
+# Prerequisites
+
+xpk uses many tool to provide all neccessary functionalities. User must install following tools:
+
+- kubectl (see [this](https://kubernetes.io/docs/tasks/tools/) link for installation.)
```

Comment:
I would reorder it by the importance to the xpk:

-python
-gcloud
- kubectl
- kueuectl

Reply:
sure thing